vpr/src/server/telegramoptions.h

```diff
@@ -34,19 +34,83 @@ class TelegramOptions {
     };
 
 public:
+    /**
+     * @brief Constructs a TelegramOptions object with the provided data and expected keys.
+     *
+     * This constructor initializes a TelegramOptions object with the given data string
+     * and a vector of expected keys. It parse data string and validate that such data has all required keys.
```

Comment:
It parses the data string and validates that ...

Reply:
fixed https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2560/commits/b454ee96877d1cc0979e20ed37fbbc360286ad7e